src/NubankClient/Model/Savings/SavingType.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NubankClient.Model
```

Comment:
@natenho I think that the namespace should represent the file path, I would use NubankClient.Model.Savings.

Reply:
You are right! My bad! Fixed